docs/detections/rules-ui-manage.asciidoc

```diff
@@ -82,6 +83,24 @@ image::images/install-prebuilt-settings.png[]
 [role="screenshot"]
 image::images/install-prebuilt-rules.png[]
 
+[float]
+[[rule-prerequisites]]
+=== Confirm rule prerequisites
+
+Many Elastic prebuilt rules are designed to work with specific Elastic integrations and data fields. These prerequisites are identified in the *Related integrations* and *Required fields* fields on a rule's details page (*Manage* -> *Rules*, then click a rule's name). *Related integrations* also displays each integration's installation status and includes links for installing and configuring the listed integrations. 
+
+Additionally, the *Setup guide* section provides guidance on setting up the rule's requirements.
+
+[role="screenshot"]
+image::images/rule-details-prerequisites.png[Rule details page with Related integrations, Required fields, and Setup guide highlighted]
+
+You can also check rules' related integrations in the *Rules* and *Rule Monitoring* tables. Click the integrations badge to display the related integrations in a popup.
+
+[role="screenshot"]
+image::images/rules-table-related-integrations.png[Rules table with related integrations popup,75%]
+
+TIP: You can hide the related integrations badge in the Rules tables. Go to *{kib}* -> *Stack Management* -> *Advanced Settings*, then turn off `securitySolution:showRelatedIntegrations`.
```

Comment:
maybe do "related integrations" with a capital at the beginning, or in code text

Reply:
I think just "integrations" to match the UI text, and bold because it's a UI label the user clicks, not text they enter.

I'll change this in a separate comment, so I can include the reference to the badge in line 97 above too.